server/global-types.ts

```diff
@@ -4,6 +4,7 @@ type Config = typeof import('../config/config-example') & AnyObject;
 
 type GroupSymbol = '~' | '&' | '#' | '★' | '*' | '@' | '%' | '☆' | '+' | ' ' | '‽' | '!';
 
+type IDOrMonitor = '' | string | string & { __isID: true} | string & {__isMonitor: true };
```

Comment:
What is the use of this type?

Reply:
No need, I'm rewriting it to be clearer.